app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextConverter.java

```diff
@@ -169,6 +185,35 @@ public String convertMarkup(String markup, Context context, boolean isExportInLi
             head += CSS_PRESENTATION_BEAMER;
         }
 
+        // Extract YAML Front Matter
+        if (!enablePresentationBeamer) {
+            if (appSettings.isMarkdownYamlDisplayEnabled() && markup.startsWith("---")) {
+                Map<String, List<String>> yamlFrontMatterMap = Collections.EMPTY_MAP;
+                yamlFrontMatterMap = extractYamlFrontMatter(markup);
+
+                if (!yamlFrontMatterMap.isEmpty()) {
+                    for (Map.Entry<String, List<String>> entry : yamlFrontMatterMap.entrySet()) {
+                        String key = entry.getKey();
+                        List<String> valueList = entry.getValue();
+                        String value = "";
+
+                        if (key.equals("tags")) {
+                            if (valueList.size() == 1) {
+                                // It's not a real tag list, but rather a string of comma-separated strings.
+                                valueList = Arrays.asList(valueList.get(0).split("(?:,\\s*)"));
+                            }
+                            Collections.sort(valueList);
+                        }
+                        for (String v : valueList) {
+                            v = v.replaceFirst("^(['\"])(.*)\\1", "$2");
+                            value += "<span class='yaml-" + key + "-item'>" + v + "</span>";
+                        }
```

Comment:
key = "date", "categories", "tags" etc right?

Reply:
Yes. The idea is to give the user the possibility to apply custom CSS rules.